Sasti/gameoff2018RhythmShooter
# #19 jungle tileset

QA

Check that the tiles provided in the jungle tileset work as expected. (collision, visuals)

Checkout the new and very nice parallax background.
